react/components/Textarea/index.js

```diff
@@ -8,6 +8,14 @@ class Textarea extends Component {
     this.state = {
       active: false,
     }
+
+    this.innerRef = createRef()
+  }
+
+  componentDidMount() {
+    if (this.props.autoFocus && this.innerRef.current) {
+      this.innerRef.current.setFocus()
```

Comment:
Error:

`TypeError: this.innerRef.current.setFocus is not a function`

Reply:
fixed